incubator/java-spring-boot2/templates/kafka/README.md

```diff
@@ -0,0 +1,169 @@
+# Kafka Template
+
+The java-spring-boot2 `kafka` template provides a consistent way of developing Spring Boot applications which connect to Kafka. This template is an extension of `default` template and uses [spring-kafka](https://spring.io/projects/spring-kafka#overview) to connect to the Kafka instance running on Kubernetes managed by [Strimzi](https://strimzi.io/) Kafka operator.
+
+The `kafka` template provides a `pom.xml` file that references the parent POM defined by the stack, dependencies that enables the Spring boot application to connect to Kafka, simple producer that publishes a message to the Kafka topic and a simple consumer that consumes the messages published on to Kafka topic by the producer. It also provides a basic liveness endpoint, and a set of unit tests that ensure enabled actuator endpoints work properly: `/actuator/health`, `/actuator/metric`, `/actuator/prometheus` and `/actuator/liveness`
+
+## Getting Started
+
+1. Create a new folder in your local directory and initialize it using the Appsody CLI, e.g.:
+
+```
+mkdir my-project
+cd my-project
+appsody init java-spring-boot2 kafka
+```
+This will initialize a Spring Boot 2 project using the kafka template.
+
+2. Once your project has been initialized you can then run your application using the following command:
+
+```
+appsody run --docker-options "--env KAFKA_BOOTSTRAP_SERVERS=${KAFKA_BOOTSTRAP_SERVERS}"
+```
+E.g:
+```
+appsody run --network kafka_default --docker-options "--env KAFKA_BOOTSTRAP_SERVERS=kafka:9092"
+```
+`DOCKER_NETWORK_NAME` is the name of the docker network in which the kafka container is running.
```

Comment:
The command above uses `kafka_default` directly as the docker network name, but was probably intended to use `$DOCKER_NETWORK_NAME`.

Reply:
I intended the line 21 to be
```
appsody run --network DOCKER_NETWORK_NAME --docker-options "--env KAFKA_BOOTSTRAP_SERVERS=${KAFKA_BOOTSTRAP_SERVERS}"
``` but missed to add `--network` part, will fix that in the next pr